content/en/docs/setup/install/multicluster/primary-remote_multi-network/index.md

```diff
@@ -174,7 +174,6 @@ $ cat <<EOF > cluster2.yaml
 apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 spec:
-  profile: remote
```

Comment:
Should these use external profile as base or no profile?

Reply:
They should be using the `external` profile, but that will require several other changes to the example, which currently relies on a local istiod for CA and injection. So, for now, I just removed the `profile=remote`, because it's deprecated. The current example continues to work with no profile (i.e., default settings). I opened #9618 to fix them to properly use `profile=external` in a future PR.